test/scaling.jl

```diff
@@ -1,4 +1,173 @@
 @testset "scaling" begin
+
+    @testset "IdentityScaling" begin
+        scaling = IdentityScaling()
+        @test scaling isa Transform
+
+        @testset "Vector" begin
+            x = [1., 2., 3.]
+            expected = [1., 2., 3.]
```

Comment:
minor nit-pick, but is it worth specifying `expected` in the case of `IdentityScaling`? since the operation is idempotent would it not be easier/clearer to say `@test Transforms.apply(x, scaling) == x` for all these tests?

Reply:
I like using expected as it ensures `x` hasn't changed